### ISSUE TITLE
Failed to install AUR helper

### ISSUE THREAD

**Issue by fzxn** [LukeSmithxyz/LARBS#464]
Error on fresh Arch installation. Followed all steps (made sure I had an internet connection, everything updated and Arch keys refreshed)

Everything goes well until yay installation then it gives this error:

> sudo: gitclonefast: command not found
> fatal: not a git repository (nor any parent on mount point /)
> Stopping at file system border (GIT_DISCOVERY_ACROSS_FILESYSTEM not set)
> Failed to install AUR helper.

(I'm sorry if this isn't the actual translation, locale was set to spanish in archinstall and the error was in spanish)

Edit: Tried to move the install script from /root to /, also uncommenting %wheel in the sudeors files and it still fails to install an AUR helper.

**Comment by LorenzoCesana** [comment]
A quick workaround is to manually install yay (https://www.makeuseof.com/install-and-use-yay-arch-linux/) and comment from line 91 to 104 (included) and line 209 (sorry for the junky workaround but I don't have time now to troubleshoot)

**Comment by jdedde** [comment]
@fzxn Clone this repo:
`git clone https://github.com/LukeSmithxyz/larbs`
Then roll it back to a working version:
`cd /path/to/larbs`
`git checkout bacb711`
It should work now.

**Comment by fzxn** [comment]
Thank you both for the workarounds. I will reinstall Arch from scratch and try a normal larbs install and then the fixes if it keeps throwing out the same errors

**Comment by LukeSmithxyz** [comment]
Might be solved in #466 now, but I still haven't had time to test the whole script.

**Comment by fzxn** [comment]
So I did a fresh Arch install and used the **archinstall** script (like I did before). On the _Additional Packages_ section I added **git** as package to install (alongside **curl** and **nano**). Then I ran the _LARBS_ script again and it worked! 
 Too bad I didn't try the workarounds mentioned but I hope they do work for other users. 

TL;DR: Installed **git** before running the LARBS script.
